### DISCUSSION TITLE
Disabling specific agent instrumentation

### DISCUSSION THREAD

**Discussion by JiZoH** [open-telemetry/opentelemetry-java-instrumentation#1986] [Q&A] [answered]
Hi
Sorry if it is a obvious one, i did not manage to find some data on this topic.

I would like to disable some agent instrumentation regarding the automatic monitoring agent using 
-Dotel.instrumentation.[id].enabled=false

Regarding this page chapter https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/suppressing-instrumentation.md#suppressing-specific-agent-instrumentation
There is a todo regarding the ids list we can use.

Is there some other documentation where i can find the ids i can use ? 
Or can it be simply resolved looking at the code ?

Thanks for your feedbacks,
cheers.

**Comment by trask** [comment, marked as answer]
Hi @JiZoH!

here's an example of where the ids are defined:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/b859485cd0981cdf1ddbba8503364f7755ee6e1a/instrumentation/jms-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jms/JmsInstrumentationModule.java#L20

if you don't find what you're looking for, let us know which instrumentation you are trying to disable

**Reply by JiZoH** [reply]
Hi @trask,
thanks a lot for your kind feedback. 
I did manage to turn off the desired verbose instrumentation.
Have a nice day.
